autoscale-runner.sls

```diff
@@ -0,0 +1,66 @@
+# This state installs a gitlab runner which is designed to act as a bastion node
+# for an autoscaling setup using docker-machine.
+# See [https://github.com/chr4/salt-docker] for some useful substates to prepare
+# the required docker base setup. In particular see states
+#   - docker
+#   - docker.login
+#   - docker.machine
+# In addition to installing the runner binary itself it supports the installation
+# of a set of pre-registered runner configurations by populating the config.toml.
+
+# Use the official Gitlab apt repository
```

Comment:
As this is used by both `runner.sls` and this file, I think its a good idea to use a seperate substate for this:

```
runner
├── autoscale.sls
├── init.sls
└── repository.sls
```

Reply:
Sounds reasonable, will take care of this